qiskit/tools/visualization/_matplotlib.py

```diff
@@ -146,6 +146,47 @@ def _registers(self, creg, qreg):
     def ast(self):
         return self._ast
 
+    def _multiqubit_gate(self, xy, fc=None, wide=True, text=None,
```

Comment:
Could you rename this to something like `_custom_multiqubit_gate()`? Just to indicate that not all multiqubit gates use this method.

Reply:
Done